app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
```

Comment:
[REQUIRED] Kindly add a `new` action to your code and routes and comments' views